pyproject.toml

```
@@ -26,7 +26,7 @@ dependencies = [
     "nest_asyncio~=1.5",
     "openai~=0.27",
     "pendulum~=2.1",
-    "prefect",
+    "prefect~=2.10.13",
```

This locks Prefect to 2.10.13 <= prefect < 2.11, I think this would be better:
```suggestion
    "prefect>=2.10.13",
```